packages/babel-preset/README.md

```diff
@@ -25,118 +25,11 @@ Then, in your Babel configuration (which should be under the `babel` key of your
-
-    - `corejs`, a number of string that will be used to set the [`corejs` version to use](https://babeljs.io/blog/2019/03/19/7.4.0#core-js-3-7646-https-githubcom-babel-babel-pull-7646)
-
-    - `useBuiltIns`, a string that is passed to the [`useBuiltIns` option of `@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env#usebuiltins)
-
-    - `transformRuntime`, a boolean (defaults to `false` and requires `typescript` to be `true`) to add the [`@babel/plugin-transform-runtime` plugin](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-runtime).
-
-    - `transformRuntimeOptions`, allows the [`@babel/plugin-transform-runtime` plugin](https://babeljs.io/docs/en/babel-plugin-transform-runtime#options) to be configured.
-
-- `@shopify/babel-preset/node`: This preset transpiles features to a specified version of Node, defaulting to the currently active version. It accepts an options object. The `modules`, `typescript`, `inlineEnv`,`debug`, `corejs` and `useBuiltIns` options do the same thing they do in `@shopify/babel-preset/web`, detailed above. You can also pass a version of Node to target during transpilation using the `version` option:
-
-  ```json
-  {
-    "babel": {
-      "presets": [
-        ["@shopify/babel-preset/node", {
-          "modules": false,
-          "version": 4
-        }]
-      ]
-    }
-  }
-  ```
-
-- `@shopify/babel-preset/react`: Adds plugins that transform React (including JSX). You can use this preset with the `@shopify/babel-preset/web` or `@shopify/babel-preset/node` configuration.
-
-  This preset accepts an options object.
-  - `pragma` : Replace the function used when compiling JSX expressions. Defaults to `React.createElement`.
-  - `pragmaFrag`: Replace the function used when compiling JSX fragment expressions. Defaults to `React.Fragment`.
-  - `transformReactConstantElements`: Adds the `@babel/plugin-transform-react-constant-elements` plugin for production environments. Defaults to `true`.
-
-  ```json
-  {
-    "babel": {
-      "presets": [
-        ["@shopify/babel-preset/react", {"pragma": "h"}]
-      ]
-    }
-  }
-  ```
-
-As noted above, you can include multiple of these presets together. Some common recipes are shown below:
-
-```js
-// A React project without any server component, using sprockets-commoner for bundling
-{
-  "babel": {
-    "presets": [
-      "@shopify/babel-preset/web",
-      "@shopify/babel-preset/react"
-    ]
-  }
-}
-
-// A Node project using Rollup to create a single bundle
-{
-  "babel": {
-    "presets": [
-      ["@shopify/babel-preset/node", {"modules": false}]
-    ]
-  }
-}
-```
-
-## Experimental Presets
-
 ## `@shopify/babel-preset/common`
 
 The `@shopify/babel-preset/common` preset is designed to be a common import that offers more options. It is a combination of the `web`, `node`, and `react` presets all in one.
```

Comment:
Gonna need to update this bit of the doc, as this is saying "this is a bit like that thing that no longer exists"